Caio_Replica/src/json/JSONReader.java

```diff
@@ -0,0 +1,44 @@
+package json;
+
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+
+import org.json.simple.JSONArray;
+import org.json.simple.JSONObject;
+import org.json.simple.parser.JSONParser;
+import org.json.simple.parser.ParseException;
+
+public class JSONReader {
+
+	public static void main(String[] args) {
+		
+		JSONParser parser = new JSONParser();
+		
+		try {
+			JSONArray replicaManagers = (JSONArray) parser.parse(new FileReader("CORBA_DFRS/src/port_config.json"));
+			
+			for (Object r : replicaManagers) {

```

Comment:
Need to parse the values into some data structure that can return the ports through a public method.